src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp

```diff
@@ -343,6 +342,10 @@ class ZynAddSubFX : public Plugin
     */
     void run(const float**, float** outputs, uint32_t frames, const MidiEvent* midiEvents, uint32_t midiEventCount) override
     {
+
```

Comment:
This code here will make the Reverse work in the Zyn lv2 plugin, which allows to use the whole Zyn inside an Lv2 Daw. However, usually, all of our effects are also available as Lv2 effect (without the Synth, only the effect) - see src/Plugin/<EffectName>. Would it be possible to add Reverse there? This would help me testing it.

Side note: The same should be done for Sympathetic.

Reply:
maybe you could write an issue for it for later.

Reply:
Done per #309 .